### PR TITLE
添加了一种登录方式（使用selenium调用Chrome进行登陆）

### PR DESCRIPTION
**Pull request by vc5**
兼容原有的browser方式，可以进行方便的切换

### CODE REVIEW

**Comment by vc5** [comment]
need fix

File: app/chrome.py [hunk 1]
```diff
@@ -126,12 +126,3 @@ def get_cookies(url)-> RequestsCookieJar:
     cookiejar = bot.cookies
     bot.quit()
     return  cookiejar
-
-
-if __name__ == '__main__':
-    config.jd = {
-        'username': '13028349489',
```
Comment (13101125) [inline app/chrome.py]: 大哥账号密码暴露了

**Comment by vc5** [comment]
谢谢




  
  
  


  
    
      
          
            
              
            
            
              Wei Vincent
            
          
            
              
                  邮箱：intelligentvincent@gmail.com
              
            
      
    
  


签名由 网易邮箱大师 定制
在2017年12月23日 01:30，13101125 写道： @13101125 commented on this pull request.



In app/chrome.py:
> @@ -126,12 +126,3 @@ def get_cookies(url)-> RequestsCookieJar:
     cookiejar = bot.cookies
     bot.quit()
     return  cookiejar
-
-
-if __name__ == '__main__':
-    config.jd = {
-        'username': '13028349489',

大哥账号密码暴露了

—You are receiving this because you modified the open/close state.Reply to this email directly, view it on GitHub, or mute the thread.


  
  




{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/CaoZ/JD-Coin","title":"CaoZ/JD-Coin","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/CaoZ/JD-Coin"}},"updates":{"snippets":[{"icon":"PERSON","message":"@13101125 commented on #37"}],"action":{"name":"View Pull Request","url":"https://github.com/CaoZ/JD-Coin/pull/37#pullrequestreview-85361425"}}}

**Comment by vc5** [comment]
@CaoZ 你这边可以帮忙操作一下吗，这个我是在不知道该怎么删除

**Pull request closed without merging**